sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/csvExpressions.scala

```diff
@@ -136,6 +139,21 @@ case class CsvToStructs(
   override def prettyName: String = "from_csv"
 }
 
+object CsvToStructs{
+  def apply(
```

Comment:
@lipzhu Please run the linter `./dev/lint-scala` and fix the style.

Reply:
@lipzhu, can you also check the style guide at https://github.com/databricks/scala-style-guide?